### ISSUE TITLE
New features (autolack, hashtag search)

### ISSUE THREAD

**Issue by lawyer88** [mifi/SimpleInstaBot#239]
Thank you very much for implementing such useful functions in the program! The only bot that is convenient and pleasant to use is your development!

** Is your request to add a feature related to any problem? Please describe it.**

Is it possible to collect the target audience by hashtags? Views, likes on photos and videos without a subscription?

**Describe the solution you would like to receive**

The ability to search for subscribers by hashtags, attracting attention to your account through views and auto-likes.
It would be ideal if it would be possible to implement a welcome message sent to new subscribers

**Describe the alternatives you have considered**
For android, there is a bot "OnClick" (Followers Assistant). This functionality responds 100% to any requests for automation of actions in Instagram.

**Comment by cgermanb** [comment]
I agree, searching for followers by hashtag would be the ideal function that the app lacks to be amazing
